dags/glam_fog.py

```diff
@@ -72,24 +72,10 @@
         email_on_retry=False,
     )
 
-    pre_import = EmptyOperator(
-        task_id="pre_import",
-    )
-
     daily_release_done = EmptyOperator(
         task_id="daily_release_done",
     )
 
-    with TaskGroup("glam_fog_external") as glam_fog_external:
```

Comment:
I removed this `ExternalTaskMarker` because most of the time I don't want to clear tasks from downstream dags